src/metasploit/msfrpc.py

```diff
@@ -1450,9 +1450,7 @@ def execute(self, **kwargs):
         if isinstance(self, ExploitModule):
             payload = kwargs.get('payload')
             runopts['TARGET'] = self.target
-            if 'DisablePayloadHandler' in runopts and runopts['DisablePayloadHandler']:
-                pass
-            elif payload is None:
+            if payload is None:
```

Comment:
Can you comment as to why the above "if" no longer applies?

Reply:
I don't know. It appears that this was an upstream change. `git blame` says it was from four years ago.
![Screen Shot 2021-09-02 at 4 25 12 PM](https://user-images.githubusercontent.com/52976633/131911448-e1b7cba2-42b1-40d0-b423-2003f0e6aa1a.png)
![Screen Shot 2021-09-02 at 4 26 14 PM](https://user-images.githubusercontent.com/52976633/131911455-b83e8a3f-2243-460d-8322-29beee646122.png)